### DISCUSSION TITLE
Periodic Table Extension

### DISCUSSION THREAD

**Discussion by yuri-kiss** [TurboWarp/extensions#702] [Show and tell]
I made a periodic table extension :).
![image](https://github.com/TurboWarp/extensions/assets/135030944/3d10d0da-5584-448a-bade-2d6264c70609)
![image](https://github.com/TurboWarp/extensions/assets/135030944/0bd60133-7dfc-4b59-9e8d-545ad9a90222)

It works sandboxed also, file: https://survexe1pc.github.io/unsafe-extensions/0znzw/PeriodicTable.js

**Comment by LilyMakesThings** [comment]
TurboWarp: Education Edition

**Reply by yuri-kiss** [reply]
yes

**Comment by NexusKitten** [comment]
Breaking Bad Utilities

**Comment by yuri-kiss** [comment]
latest update:
breaking the toolbox
![image](https://github.com/TurboWarp/extensions/assets/135030944/0be6977f-0f8d-4c2e-b3f2-24f60f7868ba)

**Reply by Flappy25** [reply]
how does the even happen?

**Reply by yuri-kiss** [reply]
it failed to parse a block but i fixed it

**Comment by yuri-kiss** [comment]
update v2, they support reporters and easier access.
![image](https://github.com/TurboWarp/extensions/assets/135030944/69628861-a1c6-4498-a6b8-7a819743a659)
![image](https://github.com/TurboWarp/extensions/assets/135030944/bd4efe72-95ca-4355-bf5a-c05823e879e0)
![image](https://github.com/TurboWarp/extensions/assets/135030944/c7e46264-82a4-4ed7-8af5-bb114ec87a9b)

**Comment by ghost** [comment]
OMG, I was just thinking of this a few days ago.

**Comment by ghost** [comment]
H2O = Water
NaCl = Salt
H2ONaCl = ???

**Reply by yuri-kiss** [reply]
how would you even get all the combinations

**Reply by ghost** [reply]
Lol Idk

**Reply by yuri-kiss** [reply]
sm

**Comment by snowboyz0825** [comment]
@SurvExe1Pc 
Hey could I get the javascript file or just straight up the text for this? I'm attempting to archive all the extensions that didn't get added and testing them all

**Comment by yuri-kiss** [comment]
its on my website.
________________________________

**Comment by yuri-kiss** [comment]
and please dont archive my extension.
________________________________
